deepchem/rl/a3c.py

```diff
@@ -263,7 +263,7 @@ def create_rollout(self):
     for i in range(self.a3c.max_rollout_length):
       if self.env.terminated:
         break
-      state = self.env.state
+      state = copy.deepcopy(self.env.state)
```

Comment:
I can change my tictactoe board and document the functions though if we decide to ask for immutability of state by the env.